source/configuration/program-detail-page.md

```diff
@@ -110,22 +110,22 @@ The Program Detail Page is divided into a Left Hand Panel and a Right Hand Panel
 
 ### Configuring the Left Hand Panel
 The left panel can display a maximum of **6** attributes as label:value pairs. If you add more than 6 attributes, **Bento will display only the first 6 attributes without a warning or error message**.
-1. Open `$(src)/bento-frontend/src/bento/programDetailData.js`.
+1. Open `$(src)/packages/bento-frontend/src/bento/programDetailData.js`.
 2. Under `leftPanel`:
-   * Add an object ({dataField: ; label: })to the `attribute` list.
+   * Add an object ({dataField: ; label: })to the `attributes` list.
     * Set the field `dataField` to the GraphQL API query that returns the data field of your choice. This will be displayed as the value of in a label:value pair.
     * Set the field `label` to a display label of your choice.
 3. Add your GraphQL API queries to `PROGRAM_DETAIL_QUERY`.
-4. You can add an embedded link to your label or your value, or both. Links can be internal or external. 
-   * To add a link to your *value* specify an internal or external link by adding a `link` attribute to your object. 
-   * To add a link to your *label* specify an internal or external link by adding a `internalLinkToLabel` or `externalLinkToLabel` attribute, respectively, to your object. 
+4. You can add an embedded link to your label or your value, or both. Links can be internal or external.
+   * To add a link to your *value* specify an internal or external link by adding a `link` attribute to your object.
```

Comment:
Remove "your" replace with "To add a link to a value", "attribute to the object"